examples/siro_sandbox/README.md

```diff
@@ -63,7 +63,7 @@ If your FPS is very low, consider this workaround. This habitat-sim commit repla
 # Command-line Options
 
 ## App State and use cases
-Use `--app-state rearrange` to run rearrange task in the sandbox app. Supported app states: `rearrange` (default), `fetch`, `socialnav`, `free_camera`. These correspond to the different use cases for the HITL tool (see corresponding implementations in `app_states/app_state_<state name>.py` files). Note, `free_camera` is special: it's not a "task" state, it's a state that lets the user control the camera instead of controlling an agent (useful for debugging and policies' behaviour visualisation). Also, `tutorial` is a special state that shows a tutorial sequence at the start of every episode to introduce the user to the scene and goals in a human-in-the-loop context. Tutorial should be followed by the task app state (`rearrange`, `fetch` or `socialnav`) and is currently supported only for the rearrange task.
+Use `--app-state rearrange` to run the rearrange task in the sandbox app. Supported app states: `rearrange` (default), `pick_throw_vr`, `socialnav`, `free_camera`. These correspond to the different use cases for the HITL tool (see corresponding implementations in `app_states/app_state_<state name>.py` files). Note, `free_camera` is special: it's not a "task" state, it's a state that lets the user control the camera instead of controlling an agent (useful for debugging and policies' behaviour visualization). Also, `tutorial` is a special state that shows a tutorial sequence at the start of every episode to introduce the user to the scene and goals in a human-in-the-loop context. Tutorial should be followed by the task app state (`rearrange`, `pick_throw_vr` or `socialnav`) and is currently supported only for the rearrange task.
 
```

Comment:
We should document the state of the 4 (?) use cases here, near the top of this file. If rearrange and socialnav are works in progress that don't currently run, that's fine, just say so. For `free_camera`, if it works, add a section for it, including keyboard controls and maybe some details like how to specify the episode dataset to load (is there a free_camera.yml config?).

For `pick_throw_vr`, just link them to your separate readme exactly as you already do (but include this here near the top of the file, not buried at the bottom).